Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -822,9 +834,9 @@ def main() -> None:  # pragma: no cover
             events = alert_events + audit_events + computer_events
             if events:
                 add_time_field(events)
+                demisto.debug(f'Sending {len(events)} events to XSIAM API')
                 send_events_to_xsiam(events=events, vendor=VENDOR, product=PRODUCT)
-                if new_last_run:
-                    demisto.setLastRun(new_last_run)
+            demisto.setLastRun(new_last_run)
```

Comment:
What's the effect of this change?

Reply:
Since JamfProtect API may return a next_page even if there are no more events, we need to update the next_run in this case as well.